lib/puppet/indirector/json.rb

```diff
@@ -52,7 +52,7 @@ def load_json_from_file(file, key)
     json = nil
 
     begin
-      json = File.read(file)
+      json = File.read(file).force_encoding(Encoding::UTF_8)
```

Comment:
I suggest we deal with the yaml terminus separately since the pain point for this ticket is specific to json (and the yaml situation is arguably more complex to fix - may be a YAGNI).